src/lib/common.js

```diff
@@ -101,10 +101,27 @@ const getConfigEnv = async options =>
     options
   )
 
+const sanitizeConfig = config => {
+  // recurse configuration objects and arrays to remove whitespace
+  const sanitize = (chunk) => {
+    return _.reduce(chunk, (result, v, k) => {
+      if (_.isObject(v) || _.isArray(v)) {
```

Comment:
What happens here if it's a numerical value?

```
"PORT": 3000
```

I think between objects, arrays, numbers, and strings we'll have all the current cases covers.

Reply:
![image](https://user-images.githubusercontent.com/9244728/73398904-4c4eca80-42b4-11ea-8c21-355909079276.png)

I think we might have some adverse side effects here. Prefer if we could branch like this:

array/object -> recurse
string -> trim
everything else (number, boolean, etc) -> return the literal value

Reply:
Great call. I'll make that adjustment. Speaking of, this method is a great candidate for unit tests. Have you considered adding a test suite?

Reply:
To keep this PR focused, I opened #46 to discuss the test suite idea.